src/code/z_fbdemo.c

```diff
@@ -40,6 +40,12 @@ Gfx sTransTileSetupDL[] = {
     gsSPEndDisplayList(),
 };
 
+#if OOT_DEBUG
+#define LOOP_INCREMENT col++
+#else
+#define LOOP_INCREMENT col++, colTex += 0x20
+#endif
```

Comment:
this one is definitely pretty sus. can you make a scratch and post it in decomp-help? maybe others will have ideas

Reply:
how can I do that for the debug version?

Reply:
I wanted to do the reverse thing for some reasons but I guess giving the scratch from debug works too https://decomp.me/scratch/oe7W2 (gonna send it on discord too)

Reply:
Hm I tried for a bit and didn't make any progress. Feel free to revert this (i.e. leave this function non-matching for retail) if you want merge the rest of the changes

Reply:
agree. its good information to know that this can make it match, but it surely is not the best solution. will just need some more people to take a crack at it

Reply:
there's probably better for `TransitionFade_Update` too so I'd suggest to close the PR for minimal bother

Reply:
that match is what mm has as well, currently. so its been tried at a bunch more, theoretically

Reply:
I'll close this then, I don't think this PR is worth if the only change is that fake match for fbdemo_fade